third_party/gtkwave_cmake/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 3.15)
+
+# find_program(CCACHE_FOUND ccache)
+# if(CCACHE_FOUND)
+#     set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
+#     set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
+# endif(CCACHE_FOUND)
+
+if (${CMAKE_VERSION} VERSION_GREATER "3.8")
+    #For cmake >= 3.9 INTERPROCEDURAL_OPTIMIZATION behaviour we need to explicitly
+    #set the cmake policy version number
+    cmake_policy(VERSION 3.9) 
+
+    # If we are using verison < 3.9 then setting INTERPROCEDURAL_OPTIMIZATION
+    # has no effect unless an Intel compiler is used
+endif()
+
+# if(NOT CMAKE_BUILD_TYPE)
+#     message(WARNING "Build type not set, falling back to Release mode.
+#         To specify build type use:
+#         -DCMAKE_BUILD_TYPE=<mode> where <mode> is Debug or Release.")
+#     set(CMAKE_BUILD_TYPE
+#         "Release"
+#         CACHE STRING "Choose the type of build, options are: Debug Release."
+#         FORCE)
+# endif(NOT CMAKE_BUILD_TYPE)
+
+project(GTKWave)
+
+if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+    set(build_dir ${PROJECT_SOURCE_DIR}/../../dbuild)
+else ()
+    set(build_dir ${PROJECT_SOURCE_DIR}/../../build)
+endif()
+
+set(gtkwave_tar "gtkwave_linux_centos7.tar.gz")
+set(download_path "https://github.com/RapidSilicon/post_build_artifacts/releases/download/v0.1/${gtkwave_tar}")
```

Comment:
isn't it private repo?

Reply:
I wondered the same thing, but it looks like the URL is public. I can dl it w/ a browser just by clicking the link https://github.com/RapidSilicon/post_build_artifacts/releases/download/v0.1/gtkwave_linux_centos7.tar.gz

Reply:
It's public. Public community can use that repo to download build artifacts. 
![image](https://user-images.githubusercontent.com/108307503/200959572-8e944521-6b7e-4d50-8f38-ab547825644d.png)